blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/CoreConfig.java

```diff
@@ -174,8 +174,13 @@
     // 3 days - this matches the TTL for our metrics_full table, we don't accept anything older than the TTL.
     BEFORE_CURRENT_COLLECTIONTIME_MS("259200000"),
     // 10 minutes
-    AFTER_CURRENT_COLLECTIONTIME_MS("600000");
+    AFTER_CURRENT_COLLECTIONTIME_MS("600000"),
 
+    // Cross-Origin Resource Sharing
+    CORS_ALLOWED_ORIGINS(""),
```

Comment:
@VinnyQ This setting should also support reqular expressions. eg if a url is prepended with ```/```, it would be treated as a regular expression. Default should be *. To enable CORS there should be an explicit boolean property  ```CORS_ENABLED``` For more reference see how ES has these settings here: https://www.elastic.co/guide/en/elasticsearch/reference/1.4/modules-http.html